### PR TITLE
Increase delay of defer calls

### PR DESCRIPTION
**Pull request by dangra**
closes #1075 and #1074

### CODE REVIEW

File: scrapy/utils/defer.py [hunk 1]
```diff
@@ -8,19 +8,25 @@
...
+    before attending pending delayed calls, so do not set delay to zero.
     """
     d = defer.Deferred()
-    reactor.callLater(0, d.errback, _failure)
+    reactor.callLater(0.1, d.errback, _failure)
     return d
 
 def defer_succeed(result):
-    """Same as twsited.internet.defer.succed, but delay calling callback until
+    """Same as twsited.internet.defer.succed but delay calling callback until
```
Comment (pablohoffman) [inline scrapy/utils/defer.py]: typo in twisted and succeed

**Comment by pablohoffman** [comment]
LGTM, let's fix the minor typos

**Comment by dangra** [comment]
fixed.

2015-05-22 19:59 GMT-03:00 Pablo Hoffman <notifications@github.com>:

**Comment by kmike** [comment]
Why not push it to master?

Hm, why did we create 1.0 branch? In past we were branching when a bug-fix release was needed, no?

**Comment by dangra** [comment]
> Why not push it to master?

do you mean pushing to master and backporting to 1.0? 

I wasn't sure what option to follow, pushing to 1.0 and porting to master or the other way. I think in the past we pushed to master and backported which makes more sense now that I think.

I will port it to master now.

> Hm, why did we create 1.0 branch? In past we were branching when a bug-fix release was needed, no?

No, in the past we created a branch for every next stable release and released bugfixes from that branch. The release procedure is described at https://github.com/scrapy/scrapy/wiki/Scrapy-release-procedure#version-and-git-branch

**Pull request merged**